gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -724,6 +724,21 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-policy-threat-slider-text">
               The default text color of <NxCode>NxPolicyThreatSlider</NxCode>.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-table-header-background">
+              The background color of the <NxCode>NxTable</NxCode> header.
+            </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-table-row-background-selected">
```

Comment:
Some of these variables have been removed

Reply:
> Some of these variables have been removed

Thanks for the reminder [44bc748](https://github.com/sonatype/sonatype-react-shared-components/pull/1072/commits/44bc7482c2d9f7bb79b85aaa7e33c0ad1fd592e4)